### DISCUSSION TITLE
Access denied

### DISCUSSION THREAD

**Discussion by eceyagman** [iterate-ch/cyberduck#15659] [Q&A]
I had no problems accessing our remote server via SFTP, but after upgrading to Sonoma today, I get the following error when trying to connect "Access denied Operation not permitted. Please verify disk permissions." However, I can open it in Terminal and access the shared folder there. Can anyone help with troubleshooting?

Thank you!

**Comment by tatze96** [comment]
The mounting process failed as the macOS upgrade changed the privacy settings, causing the custom mount location to be inaccessible.

**Comment by GaborTorma** [comment]
<img width="424" alt="image" src="https://github.com/user-attachments/assets/56743bb0-6058-4182-91a7-f631e8af7ffe" />

I have the same issue, after buying Mountain Duck via App Store.
I grant full disk access for Mountain Duck, but it doesn't solve the problem.
I can't change Mount location also:
<img width="1049" alt="image" src="https://github.com/user-attachments/assets/8196b39b-3cb4-443a-be38-ee8db640fada" />

**Reply by janzheng** [reply]
it's been a year did you figure this out

**Reply by dkocher** [reply]
When running Mountain Duck installed from the Mac App Store, make sure to reset the custom mount location to the default.

**Reply by joakimmag** [reply]
@dkocher where do you do that?

**Reply by dkocher** [reply]
You can reset the sync cache folder to the default using _Terminal.app_

```
defaults delete io.mountainduck fs.sync.cache.directory 
```
